spec/lib/publishing_api_payload_spec.rb

```diff
@@ -156,6 +156,15 @@
       expect(payload[:links][:government]).to eq [government.content_id]
     end
 
+    it "includes public_updated_at if the edition has backdated_to and is a first edition" do
+      date = Time.zone.now.yesterday
+      revision = build(:revision, backdated_to: date)
+      edition = build(:edition, revision: revision, number: 1)
+      payload = described_class.new(edition).payload
+
+      expect(payload).to match a_hash_including(public_updated_at: date)
+    end
```

Comment:
I'm not sure where this has come from. I wonder if it's removed code that has slipped in from a rebase?